docker-compose.yml

```diff
@@ -107,7 +107,7 @@ services:
 
   db:
     container_name: koku_db
-    image: postgres:10.6
+    image: postgres:latest
```

Comment:
Should we use the 12 tag explicitly so this doesn't bump on us before we're ready?

Reply:
Yeah. I had pulled an image labeled 12 that ended up a minor version behind what I knew was available. So I pulled out too big of a hammer.